dockerfiles/executor_image/data/registries.conf

```diff
@@ -7,15 +7,3 @@ prefix = "docker.io"
 location = "docker.io"
 [[registry.mirror]]
 location="mirror.gcr.io"
-
-# 2020-10-27 a number of images are not present in gcr.io, and podman barfs spectacularly when trying to fetch them. We've
-# hand-copied those to quay, using skopeo copy --all ...
-[[registry]]
-prefix="docker.io/library"
-location="quay.io/libpod"
-
-# 2021-03-23 these are used in buildah system tests, but not (yet?) listed in the global shortnames.conf.
-[aliases]
-busybox="docker.io/library/busybox"
-ubuntu="docker.io/library/ubuntu"
-php="docker.io/library/php"
```

Comment:
cc @luluz66 any idea if this change would break any existing use cases?

Reply:
I think this was taken as-is from an older version of https://github.com/containers/podman/blob/main/test/registries.conf

I think the removed part is only relevant to RedHat / Podman-specific setup.

Reply:
This looked fine to me; but probably run a few builds in dev to make sure we didn't break anything.

Reply:
Can you revert this specific diff since it's unrelated to the PR? We can revisit later if it's needed, but would prefer to avoid unnecessary churn here.

Reply:
+1 -- this looks fine to move to another CL

Reply:
Reverted that chunk. Will move to another PR

Reply:
created https://github.com/buildbuddy-io/buildbuddy/pull/4350